x2gbfs/providers/moqo.py

```diff
@@ -0,0 +1,251 @@
+            self._extract_from_vehicles(elem, gbfs_vehicles_map, gbfs_vehicle_types_map)
+
+        return gbfs_vehicle_types_map, gbfs_vehicles_map
+
+    @staticmethod
+    def _format_timestamp(timestamp: int) -> str:
+        return datetime.fromtimestamp(timestamp).strftime('%Y-%m-%dT%H:%M')
+
+    def _get_paginated(self, element_type: str, params: Optional[dict[str, str]] = None):
+        url = self.api_url + element_type
+        headers = {'Authorization': f'Bearer {self.api_token}', 'accept': 'application/json'}
+        page = 1
+        while True:
+            request_params = dict(params) if params is not None else {}
+            request_params['page[number]'] = str(page)
+            json_response = get(url, params=request_params, headers=headers).json()
+
+            if 'data' in json_response:
+                for elem in json_response['data']:
+                    yield elem
+            else:
+                logger.warning(f'Response to {url} did not contain {element_type}')
+                logger.warning(json_response)
+                return
+
+            if json_response.get('pagination', {}).get('next_page'):
+                page += 1
+            else:
+                break
+
+    def _extract_from_parkings(
+        self,
+        elem: dict[str, Any],
+        gbfs_station_infos_map: dict[str, dict],
+        gbfs_station_status_map: dict[str, dict],
+        default_last_reported: int,
+    ) -> None:
+        station_id = elem['id']
+
+        center = elem['center']
+        station = {
+            'lat': center['lat'],
+            'lon': center['lng'],
+            'name': elem['name'],
+            'station_id': str(elem['id']),
+            'addresss': elem['street'] + ' ' + elem['street_number'],
+            'post_code': elem['zipcode'],
+            '_city': elem['town'],  # Non-standard
+            'rental_methods': ['key'],
+        }
+
+        if elem['capacity_max']:
+            station['capacity'] = elem['capacity_max']
+
+        gbfs_station_infos_map[station_id] = station
+
+        gbfs_station_status_map[station_id] = {
+            'num_bikes_available': 0,
+            'vehicle_types_available': {},
+            'is_renting': True,
+            'is_installed': True,
+            'is_returning': True,
+            'station_id': str(elem['id']),
+            'last_reported': default_last_reported,
+        }
+
+    def _extract_from_vehicles(
+        self,
+        vehicle: dict[str, Any],
+        vehicles: dict[str, Any],
+        vehicle_types: dict[str, Any],
+    ) -> None:
+        vehicle_type_id = self._extract_vehicle_type(vehicle_types, vehicle)
+        vehicle_id = vehicle['id']
+        gbfs_vehicle = {
+            'bike_id': str(vehicle_id),
+            'is_reserved': vehicle['available'] is not True,
+            'is_disabled': False,
+            'vehicle_type_id': vehicle_type_id,
+            'license_plate': vehicle['license'].split('|')[0].strip(),
```

Comment:
we get some invalid license plates, e.g., TBB**-**S 298 E

Reply:
Could you contact Stadtwerk Tauberfranken to address this at the root?